packages/admin/schemas/documents/product.js

```diff
@@ -0,0 +1,83 @@
+export default {
+  name: "product",
+  type: "document",
+  title: "Products",
+  fieldsets: [
+    {
+      title: "SEO & metadata",
+      name: "metadata",
+    },
+  ],
+  initialValue: {
+    includeInSitemap: true,
+    disallowRobots: false,
+  },
+  fields: [
+    {
+      name: "title",
+      type: "string",
+      title: "Title",
+    },
+    {
+      title: "Slug",
+      name: "slug",
+      type: "slug",
+      options: {
+        source: "title",
+        maxLength: 200, // will be ignored if slugify is set
+        slugify: (input) =>
+          input.toLowerCase().replace(/\s+/g, "-").slice(0, 200),
+      },
+    },
+    { name: "hero", type: "hero", title: "Hero" },
+    { name: "why", type: "why", title: "Why Livpeeer.com" },
```

Comment:
@FullStackTiger We don't have to change now, but in a follow up PR we should generalize / abstract the name of this field type and component. i.e. `why` -> `listItem`